Paintroid/res/values/string.xml

```diff
@@ -130,7 +130,7 @@
 
     -->
     <string name="crop_algorithm_finish_text">to crop tap</string>
-    <string name="crop_nothing_to_corp">nothing to crop</string>
+    <string name="crop_nothing_to_crop">nothing to crop</string>
```

Comment:
this is the file where you can change the string name. All the other string.xml files have to be changed via CrowedIn.